crutest/src/main.rs

```diff
@@ -1068,6 +1062,23 @@ async fn main() -> Result<()> {
         Workload::Version => {
             panic!("This case handled above");
         }
+        Workload::WFR => {
+            println!("Run Write-Flush-Read random IO test");
+            let count = {
+                if opt.count == 0 {
+                    10
+                } else {
+                    opt.count
+                }
+            };
+            write_flush_read_workload(&guest, count, &mut region_info).await?;
+        }
+    }
+
+    if opt.verify_at_end {
```

Comment:
Would it make sense to have `verify_at_end` default to `true`? 
 
For example, `balloon_workload` used to unconditionally call `verify_volume`; that call has been removed, so we'll now only do the verification here if the flag is set.  Making the flag default to true would preserve our current behavior, but still let you turn it off with `--verify-at-end=false`

Reply:
I don't want all the tests to verify-at-end though, as that can add a bunch of run time to, like the perf tests or other larger tests.  

How about I go add the `--verify-at-end` for the one place in CI that used balloon to preserve that behavior and let any other test that wants it can add it.

Reply:
Okay, that's fine by me.  It looks like `perf` is also used in CI (in `test_perf.sh`); up to you whether you want to keep previous behavior there as well.

Reply:
Yeah, if perf was doing a final verify then I think that's actually the wrong behavior.  The perf test
should just be testing perf (and it's already rather long for CI).

I did just update the count -> Option suggestion you made.